docs/variants/msi_z790/hardware-matrix.md

```diff
@@ -0,0 +1,31 @@
+# Hardware configuration matrix
+
+## Introduction
+
+This document describes the hardware configuration used for validation of the
+coreboot port on the MSI PRO Z790-P workstation. For additional information,
+please check the [FAQ section](./faq.md).
+
+## MSI PRO Z790-P, SN: 601-7E06-030B2210001398
+
+| Component                      | Description                                 |
+|--------------------------------|---------------------------------------------|
+| **CPU**                        | Intel Core i5-13600K 3.5                    |
+|                                | Noctua NH-U12S CPU Cooler                   |
+| **RAM**                        | Slot 1: Kingston KF436C17BBK4/32            |
+|                                | Slot 2: Kingston KF436C17BBK4/32            |
+|                                | Slot 3: Kingston KF436C17BBK4/32            |
+|                                | Slot 4: Kingston KF436C17BBK4/32            |
```

Comment:
Kignston KF436C17BBK4 is DDR4 memory, our Z790-P has DDR5 slots, please fix

Reply:
I changed the RAM to correct model.